dymos/phase/phase.py

```diff
@@ -1257,9 +1257,9 @@ def add_timeseries_output(self, name, output_name=None, units=_unspecified, shap
         ----------
         name : str, or list of str
             The name(s) of the variable to be used as a timeseries output, or a mathematical expression to be used
-            as a timeseries output. If a name, it be one of 'time', 'time_phase', one of the states, controls,
-            control rates, or parameters, in the phase, the path to an output variable in the ODE, or a glob pattern
-            matching some outputs in the ODE.
+            as a timeseries output. If a name, it be one of the integration variable, 'time_phase', one of the states,
```

Comment:
"If a name, it be one of" -> "If a name, it should be one of"? or maybe a word is missing in here